schedule/sles4sap_offline_gnome_staging.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name:           sles4sap_offline_gnome
```

Comment:
The test suite is applied for both Sle12 and Sle15. As I understand, this is the schedule only for Sle15. 

In order to be consistent with the other changes from this PR, I guess it would be great to support both versions. Examples of jobs: [SLE12](https://openqa.suse.de/tests/2811715), [SLE15](https://openqa.suse.de/tests/2811633).

Reply:
sles4sap_offline_gnome was not listed in the Test result overview. @rwx788 what s true for this?